lbz/dev/server.py

```diff
@@ -6,16 +6,12 @@
 import urllib.parse
 from abc import ABCMeta, abstractmethod
 from http.server import BaseHTTPRequestHandler, ThreadingHTTPServer
-from os import environ
 from threading import Thread
 from typing import Tuple, Type, Union
 
 from lbz.dev.misc import APIGatewayEvent
 from lbz.resource import Resource
 
-if environ.get("LBZ_DEBUG_MODE") is None:
-    environ["LBZ_DEBUG_MODE"] = "true"
```

Comment:
It makes a little sense to have it in the dev server 🤔 

At least I remember you added it to always enable debug mode here.